.mailmap

```diff
@@ -186,6 +186,7 @@ Anthony Sottile <asottile@umich.edu>
 Anton Akhmerov <anton.akhmerov@gmail.com>
 Anubhav Gupta <anubhav.gupta.cse19@itbhu.ac.in>
 Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
+Anurag Bhat <90216905+faze-geek@users.noreply.github.com>
 Anurag Bhat <bhat.1@iitj.ac.in> faze-geek <90216905+faze-geek@users.noreply.github.com>
 Anurag Bhat <bhat.1@iitj.ac.in> faze-geek <bhat.1@iitj.ac.in>
```

Comment:
This is adding `Anurag Bhat <90216905+faze-geek@users.noreply.github.com>` as a separate author from `Anurag Bhat <bhat.1@iitj.ac.in>`. The second line should be all that is needed and so the first line should be removed.

Reply:
Are you sure about that .Cannot be seen in commit history now but I correctly remember the authors check was failing without this addition ! Can you recheck if this is actually unnecessary ?

Reply:
Here is the error message from #22827 (a pr with a recent commit after changing my username in github .The failed check is still present on that pr if you would like to see )
```
This author is not included in the .mailmap file:
Anurag Bhat <[9](https://github.com/sympy/sympy/runs/5328406008?check_suite_focus=true#step:6:9)0216905+faze-geek@users.noreply.github.com>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.
```

Reply:
@oscarbenjamin A bit confusing because after removing `Anurag Bhat <90216905+faze-geek@users.noreply.github.com>` and running `python mailmap_check.py` it does not give any errors .

Reply:
I just checked out this PR and deleted the line and then ran the `mailmap_check.py` script:
```console
$ git diff
diff --git a/.mailmap b/.mailmap
index 7526a14..80034d3 100644
--- a/.mailmap
+++ b/.mailmap
@@ -186,7 +186,6 @@ Anthony Sottile <asottile@umich.edu>
 Anton Akhmerov <anton.akhmerov@gmail.com>
 Anubhav Gupta <anubhav.gupta.cse19@itbhu.ac.in>
 Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
-Anurag Bhat <90216905+faze-geek@users.noreply.github.com>
 Anurag Bhat <bhat.1@iitj.ac.in> faze-geek <90216905+faze-geek@users.noreply.github.com>
 Anurag Bhat <bhat.1@iitj.ac.in> faze-geek <bhat.1@iitj.ac.in>
 Anurag Sharma <anurags92@gmail.com> <sharmaan@iitk.ac.in>
$ bin/mailmap_check.py 
No changes needed in .mailmap

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Siddhant Jain <siddhantashoknagar@gmail.com>
```
So that line is not needed at least not for the commits in this PR. Even if it was needed it should be:
```
Anurag Bhat <bhat.1@iitj.ac.in> Anurag Bhat <90216905+faze-geek@users.noreply.github.com>
```

Reply:
Yes instead of deleting it I'll replace it with what you have mentioned. Thanks !

Reply:
I have made the change now ! I get the essence of how a it was being treated as a separate author .Rest all seems fine so could be merged after all checks pass .

Reply:
@oscarbenjamin All checks have passed now , I guess it is ready to go in .Kindly have a look !